Blaise-Shyaka/My-Mentor
# [finishes #170308195] responsive welcome home page

# What does this PR do?

This is the home page of the web application

# Tasks to be completed

- Create a welcome page with sign-in and sign-up buttons
- Make it responsive

# How should this be manually tested

Go into the views directory, open index.html file. Resize your browser to verify its responsiveness.
